README.md

```diff
@@ -1,29 +1,41 @@
 # Subcontracting
 
-Adds an object for information about the parts of the contract that the supplier will subcontract to third parties.
+Adds objects for information about the terms governing subcontracting and the parts of the contract that tenderers and suppliers will subcontract to third parties.
 
 ## Guidance
 
+If you are using the [Lots extension](https://extensions.open-contracting.org/en/extensions/lots/master/), [follow its guidance](https://extensions.open-contracting.org/en/extensions/lots/master/#usage) on whether to use `tender.lots` fields or `tender` fields.
+
 If the percentage of the contract value that is subcontracted is an exact number and not a range, set `minimumPercentage` and `maximumPercentage` to the same number.
 
 ## Legal context
 
-In the European Union, this extension's fields correspond to [eForms BG-709 (Second Stage), BT-65, BT-64, BT-729](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/) and [article 21 of directive 2009/81/EC](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32009L0081&from=EN#d1e2623-76-1). See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
+In the European Union, this extension's fields correspond to [article 21 of directive 2009/81/EC](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32009L0081&from=EN#d1e2623-76-1) and the following [eForms business terms](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/):
+
+* BT-773 (Subcontracting)
+* BT-553 (Subcontracting Value)
+* BT-554 (Subcontracting Description)
+* BT-555 (Subcontracting Percentage)
+* OPT-301 (Main Contractor ID Reference, Subcontractor ID Reference)
+* BT-64 (Subcontracting Obligation Minimum)
+* BT-65 (Subcontracting Obligation)
+* BT-729 (Subcontracting Obligation Maximum)
```

Comment:
This is a little verbose. Maybe say  "business terms in BG-180 (Subcontracting) and BG-711 (Contract Terms)"

Reply:
Done in e05474a76f49b07a121c49747ccba5e8afd9345e